annotation-tests/src/integration-test/java/eu/europeana/annotation/tests/web/AnnotationSearchIT.java

```diff
@@ -432,9 +475,9 @@ protected Annotation searchLastCreated(String query) throws Exception {
     return anno;
   }
 
-  protected AnnotationPage search(String bodyValue, SearchProfiles profile, String limit)
+  protected AnnotationPage search(String bodyValue, String profile, String limit)
       throws Exception {
-    AnnotationPage annPg = searchAnnotations(bodyValue, null, WebAnnotationFields.CREATED, "desc",
+    AnnotationPage annPg = searchAnnotations(bodyValue, null, null, WebAnnotationFields.CREATED, "desc",
```

Comment:
shouldn't be the profile passed over to the search method at this place?

Reply:
It is passed in the before last parameter.

Reply:
it is passed null instead of profile ...